packages/terra-application-utility/package.json

```diff
@@ -32,7 +32,15 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-base": "^3.0.0"
+    "terra-app-delegate": "^2.0.0",
```

Comment:
Do we need to add some of these as peer?

Reply:
terra-app-delegate at least should be defined as a peer dependency.